pubnub-memberships/src/integrationTest/kotlin/com/pubnub/entities/MembershipIntegTest.kt

```diff
@@ -61,13 +63,13 @@ class MembershipIntegTest {
         createSpace(SPACE_ID, SPACE_NAME)
         createSpace(SPACE_ID_02, SPACE_NAME_02)
         val spaceIdWithCustomList: List<Membership.PartialWithSpace> =
```

Comment:
What about changing name of "spaceIdWithCustomList" to accommodate to new type?

Reply:
I thought I got it all :)